src/NClient.Tools/NClient.DotNetTool/Program.cs

```diff
@@ -51,33 +52,75 @@ public static async Task<int> Main(string[] args)
                         (FacadeOptions.GenerationOptions generateOptions) => HandleFacadeGenerationOptions(generateOptions), 
                         facadeErrors => HandleErrors(facadeParserResult, facadeErrors, showGreeting: false));
                 },
+                (ClientOptions _) =>
+                {
+                    var clientParserResult = parser.ParseArguments<ClientOptions.GenerationOptions, int>(args.Skip(1));
+                    return clientParserResult.MapResult(
+                        (ClientOptions.GenerationOptions generateOptions) => HandleFacadeGenerationOptions(generateOptions), 
+                        facadeErrors => HandleErrors(clientParserResult, facadeErrors, showGreeting: false));
+                },
                 errors =>
                 {
                     var errorArrays = errors as Error[] ?? errors.ToArray();
                     var errorTag = errorArrays.First().Tag;
-                    if (errorTag is not ErrorType.HelpRequestedError && errorTag is not ErrorType.UnknownOptionError)
+                    if (args.Length == 0 || errorTag is not ErrorType.HelpRequestedError && errorTag is not ErrorType.UnknownOptionError)
```

Comment:
Again. Complex conditions in separate variable is more readable.

Reply:
Done